lib/ocpp/v201/evse.cpp

```diff
@@ -311,7 +311,8 @@ void Evse::check_max_energy_on_invalid_id() {
         if (opt_energy_value.has_value() and active_energy_import_start_value.has_value()) {
             auto charged_energy = opt_energy_value.value() - active_energy_import_start_value.value();
 
-            if (charged_energy > static_cast<float>(max_energy_on_invalid_id.value())) {
+            // TODO float compare with epsilon
```

Comment:
Hi @rinzevdwalAlfen , can you do this comparison using an epsilon as part of this PR?

Reply:
Hello @Pietfried, I have added epsilon to the statement.